### PR TITLE
3.5 Pipeline

### PR DESCRIPTION
**Pull request by akofink**

### CODE REVIEW

File: pipelines/pipeline_katello_35.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      - foreman-installer-katello
+  roles:
+    - foreman_proxy_content
+    - foreman_installer
+
+- hosts: pipeline-katello-3.5-centos7
+  become: true
+  tasks:
+    - name: 'Restart services'
+      command: 'katello-service restart'
```
Comment (ekohl) [inline pipelines/pipeline_katello_35.yml]: Why is this here? Is it required or part of a test?

Comment (akofink) [reply pipelines/pipeline_katello_35.yml]: I agree. Removed!

**Comment by ehelms** [comment]
I will need to update PUlp to 2.13.4 in order to fix the current breakage.

**Comment by akofink** [comment]
Ready for merge, I believe.
